src/Compilers/Core/Desktop/CommandLine/CommonCommandLineArguments.cs

```diff
@@ -328,13 +328,51 @@ public IEnumerable<AnalyzerReference> ResolveAnalyzerReferences(IAnalyzerAssembl
         {
             foreach (CommandLineAnalyzerReference cmdLineReference in AnalyzerReferences)
             {
-                yield return ResolveAnalyzerReference(cmdLineReference, analyzerLoader)
-                    ?? (AnalyzerReference)new UnresolvedAnalyzerReference(cmdLineReference.FilePath);
+                string resolvedPath = ResolveAnalyzerReferencePath(cmdLineReference);
+
+                if (resolvedPath == null)
+                {
+                    yield return new UnresolvedAnalyzerReference(cmdLineReference.FilePath);
+                }
+
+                Assembly assembly = null;
+                try
+                {
+                    assembly = analyzerLoader.LoadFromPath(resolvedPath);
+                }
+                catch (Exception)
+                {
```

Comment:
Is this resolve error eventually reported to the user?  I would expect something along the lines of:

> Unable to load the analyzer assembly <path>.  Error: <exception info>